src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;

```

Comment:
You have a specific enum to represent colors, please use it here instead of the String.